### PR TITLE
minor README update and switched Sacramento font to Cookie

### PR DESCRIPTION
**Pull request by sonofab1rd**
@cdierkens 

Can you have a look? Just to make sure I didn't overlook anything. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,12 @@ Sanity.io data studio for Bluebird Cabinetry and Design
...
 
-By default `studio` is configured to use the `development` dataset in your local environment. Be sure `development.env` in the `web` package is set to build the `development` dataset.
+By default `studio` is configured to use the `development` dataset in your local environment. Be sure `.env.development` in the `web` package is set to build the `development` dataset.
+
+Example `web/.env.development`
 
-Example `web/development.env`
 ```
-# web/development.env
+# web/.env.development
```
Comment (cdierkens) [inline README.md:22]: good find

**Comment by cdierkens** [comment]
@sonofab1rd  The latest tailwind required postCSS v8 while other library have v7 as dependencies. This could be part of the build issue (if the NODE_VERSION doesn't solve it)

**Comment by sonofab1rd** [comment]
@cdierkens 
The NODE_VERSION addition got the preview build to work. But, it looks like not all the checks passed. 
https://community.netlify.com/t/node-12-environment-update/6170/2

**Comment by cdierkens** [comment]
> @cdierkens
> The NODE_VERSION addition got the preview build to work. But, it looks like not all the checks passed.
> https://community.netlify.com/t/node-12-environment-update/6170/2

Looks like the unit test actions just failed to actually start. GH Actions is buggy like that sometimes.

**Pull request merged**